### PR TITLE
Add `remaster` table class to `_globals.css`

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: src/styles/_globals.scss [hunk 1]
```diff
@@ -135,6 +135,47 @@ table.pf2-table {
...
+        user-select: text;
+
+        thead,
+        tbody,
+        th,
+        td {
+            box-sizing: border-box;
+            user-select: text;
+            word-break: auto-phrase;
+        }
```
Comment (stwlam) [inline src/styles/_globals.scss:152]: Remove anything not theme-specific

Comment (ammalagonc) [reply src/styles/_globals.scss]: Check when you get a moment.

Thanks!

File: src/styles/_globals.scss [hunk 2]
```diff
@@ -135,6 +135,36 @@ table.pf2-table {
...
+        td {
+            border: solid 1px var(--color-border-light-tertiary);
+            text-align: center;
+        }
+
+        th {
+            background-color: #002a17;
+            color: var(--text-light);
+            font-weight: bold;
+            padding: 0.5rem 0.25rem;
```
Comment (stwlam) [inline src/styles/_globals.scss:161]: Use space variables (e.g.,  `padding: var(--space-8) var(--space-4);`)

File: src/styles/_globals.scss [hunk 3]
```diff
@@ -129,11 +129,41 @@ table.pf2-table {
...
+        th {
+            background-color: #002a17;
+            color: var(--text-light);
+            font-weight: bold;
+            padding: var(--space-8) var(--space-4);
+        }
+
+        td {
+            padding: 0.25rem;
+        }
```
Comment (stwlam) [inline src/styles/_globals.scss:166]:
```suggestion
```

File: src/styles/_globals.scss [hunk 4]
```diff
@@ -129,11 +129,41 @@ table.pf2-table {
...
+        th,
+        td {
+            border: solid 1px var(--color-border-light-tertiary);
+            text-align: center;
+        }
+
+        th {
+            background-color: #002a17;
+            color: var(--text-light);
+            font-weight: bold;
```
Comment (stwlam) [inline src/styles/_globals.scss:160]:
```suggestion
```
Remember that you're just overriding the thing syou want to change.

File: src/styles/_globals.scss [hunk 5]
```diff
@@ -129,11 +129,41 @@ table.pf2-table {
...
 
     td {
-        padding: 0.25rem;
+        padding: var(--space-4);
+    }
+
+    &.remaster {
+        color: var(--text-dark);
+        font-size: var(--font-size-13);
+
```
Comment (stwlam) [inline src/styles/_globals.scss:142]:
```suggestion
```

**Review by stwlam — CHANGES_REQUESTED** [review]

File: src/styles/_globals.scss [hunk 6]
```diff
@@ -129,11 +129,32 @@ table.pf2-table {
...
+        th,
+        td {
+            border: solid 1px var(--color-border-light-tertiary);
+            text-align: center;
+        }
+
+        th {
+            background-color: #002a17;
+            padding: var(--space-8) var(--space-4);
```
Comment (stwlam) [inline src/styles/_globals.scss:156]: omg

Comment (stwlam) [inline src/styles/_globals.scss:156]:
```suggestion
```

File: src/styles/_globals.scss [hunk 7]
```diff
@@ -129,11 +129,32 @@ table.pf2-table {
...
+        }
+
+        tr:nth-child(even) {
+            background-color: #f5efe0;
+        }
+
+        th,
+        td {
+            border: solid 1px var(--color-border-light-tertiary);
+            text-align: center;
```
Comment (stwlam) [inline src/styles/_globals.scss:151]:
```suggestion
            border-border: var(--color-border-light-tertiary);
```

**Review by stwlam — APPROVED** [review]

**Pull request merged**